adoc/quickstart3_chap_suma_salt_gs.adoc

```diff
@@ -136,7 +134,7 @@ Find more functions which can be called on minions by running:
 salt '*' sys.doc
 ----
 +
-A full list of callable functions are located here: https://docs.saltstack.com/en/2015.8/ref/modules/all/index.html
+A full list of callable functions are located here: https://docs.saltstack.com/en/latest/ref/modules/all/index.html
```

Comment:
Better use our standard phrase:

> For more information about a full list of ..., see https://... .

Reply:
"more information about" doesn't make sense, so I've gone with "For a full list, see" so we have the right structure.

https://github.com/SUSE/doc-susemanager/pull/311